rpc_util.go

```diff
@@ -297,21 +297,34 @@ func (o FailFastCallOption) before(c *callInfo) error {
 func (o FailFastCallOption) after(c *callInfo, attempt *csAttempt) {}
 
 // OnFinish returns a CallOption that configures a callback to be called when
-// the call completes.
+// the call completes. The error passed to the callback is the status of the
+// RPC, and may be nil. Must only be called once. This is mainly used in the
```

Comment:
Not "Must only be called once", but "The onFinish callback provided will only be called once by gRPC." (e.g.)

Reply:
Switched to that. Isn't this implied though? And it feels correct to only be called once because it's the event when the RPC finishes.